src/PostRector/Application/PostFileProcessor.php

```diff
@@ -27,20 +27,34 @@ final class PostFileProcessor implements ResetableInterface
      */
     private array $postRectors = [];
 
+    /**
+     * @param PostRectorInterface[] $additionalPostRectors
+     */
     public function __construct(
         private readonly Skipper $skipper,
         private readonly UseAddingPostRector $useAddingPostRector,
         private readonly NameImportingPostRector $nameImportingPostRector,
         private readonly ClassRenamingPostRector $classRenamingPostRector,
         private readonly DocblockNameImportingPostRector $docblockNameImportingPostRector,
         private readonly UnusedImportRemovingPostRector $unusedImportRemovingPostRector,
-        private readonly RenamedClassesDataCollector $renamedClassesDataCollector
+        private readonly RenamedClassesDataCollector $renamedClassesDataCollector,
+        private array $additionalPostRectors
     ) {
     }
 
     public function reset(): void
     {
         $this->postRectors = [];
+        $this->additionalPostRectors = [];
+    }
+
+    /**
+     * @param PostRectorInterface[] $postRectors
+     * @api used in tests to update the list of additional post rectors
+     */
+    public function refreshAdditionalPostRectors(array $postRectors): void
```

Comment:
This is used to inject the list of additional post rectors when running the tests